src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -208,6 +208,7 @@ def load_arguments(self, _):
         c.argument('enable_private_cluster', action='store_true')
         c.argument('nodepool_tags', nargs='*', validator=validate_nodepool_tags, help='space-separated tags: key[=value] [key[=value] ...]. Use "" to clear existing tags.')
         c.argument('enable_managed_identity', action='store_true')
+        c.argument('assign_identity', type=str, validator=validate_assign_identity)
```

Comment:
Is there a way to remove identity?

Reply:
There is a preview feature which can switch the identity, but at least one and only one identity should be associated with the cluster.